internal/pkg/config/model.go

```diff
@@ -66,3 +69,7 @@ type RedisConfig struct {
 	PoolSize     int
 	IdleTimeout  time.Duration
 }
+
+type BlobConfig struct {
```

Comment:
nit: add a comment

Reply:
Thank you for pointing out.
I'll add a comment to that.